Documentation/teaching/lectures/syscalls.rst

```diff
@@ -372,7 +372,8 @@ With VDSO the system call interface is decided by the kernel:
    :level: 2
 
    * a stream of instructions to issue the system call is generated by
-     the kernel in a special memory area (as en ELF .so)
+     the kernel in a special memory area (in memory it is formated as
```

Comment:
Maybe "(formatted as an ELF shared object)" is better? Since it is in memory, it is not a file :)